### ISSUE TITLE
ID: 251877 [Screen reader - Home]: Screen reader does not announce tooltip details for the "Time range zoom out" button

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#66471]
### User Experience
 People with low vision and cognitive limitations will not be accessing the instructions for using the content. If the user cannot perceive the data of tool tip of components and no additional information is provided by screen reader.

### Repro steps
Open #Url: https://play.grafana.org/   Sign in with valid credentials. "Home" page appears. Navigate the page. navigate to the header section. Verify the issue.

### Actual result
Screenreader does not announce the tooltip details (Time range zoom out Ctrl + Z) for the "Time range zoom out" button present under the header section on the home page even with the narrator command (caps lock + 0) it is announce like {no advance information available).

### Expected result:
Screen reader should announce the tooltip details (Time range zoom out Ctrl + Z) for the "Time range zoom out" button present under the header region on the home page narrator should announce "Time range zoom out Ctrl + Z" (shortcut to announce the tool-tip command is caps lock + 0).

### Additional information
No tooltips are actually readout by the screen reader. It happens that most buttons have an `aria-label` that matches the tooltip so it _appears_ the tooltip is read out but it isn't. The solution to this should be that tooltips must match the `aria-label` provided.

### Severity
`SEV 2`

### Area
TimeRangePicker / Tooltips (Saga Design System)

### MS Sheet ID
[251877](https://docs.google.com/spreadsheets/d/1kB-Sz3lvfje93HLVIx4UhDvwgU-J7k1PVN70ryQkc4M/edit#gid=888389206&range=A2)
This also covers 252066.

https://user-images.githubusercontent.com/6906380/231777813-be08d2b3-fd87-4812-b9e2-c5c1559154a4.mov

UHG Audit CP ID: 1.112

**Comment by ckbedwell** [comment]
Getting addressed in: https://github.com/grafana/grafana/issues/59350

**Comment by ckbedwell** [comment]
Fixed in https://github.com/grafana/grafana/pull/79465 🍾
